lib/zero_phoenix/account/account.ex

```diff
@@ -36,6 +36,31 @@ defmodule ZeroPhoenix.Account do
 
   """
   def get_person!(id), do: Repo.get!(Person, id)
+  def get_person(id), do: Repo.get(Person, id)
+
+  @doc """
+  Gets a list of persons.
+
+  ## Examples
+
+      iex> get_persons!([123])
+      {:ok, [%Person{}]}
+
+      iex> get_persons!([123, 456])
+      {:ok, [%Person{}]}
+
+  """
+  def get_persons(ids) do
```

Comment:
I recommend renaming `get_persons` to `get_people` that takes a list.  For example,

```elixir
@doc """
  Gets people from given list ids.

  Returns `[]` if ids is empty or if ids don't exist.

  ## Examples

      iex> get_people([123])
      [%Person{}, ...]

      iex> get_people([456, 789])
      []

  """
  def get_people(ids \\ []) do
    from(p in Person, where: p.id in ^ids)
    |> Repo.all()
  end
```